packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/TxDetailModal.tsx

```diff
@@ -14,12 +14,16 @@ import { RequiredKey } from '@trezor/type-utils';
 import { useSelector } from 'src/hooks/suite';
 import { Translation } from 'src/components/suite';
 import { Account, WalletAccountTransaction } from 'src/types/wallet';
+import { RbfContext, useRbf } from 'src/hooks/wallet/useRbfForm';
 
 import { AdvancedTxDetails, TabID } from './AdvancedTxDetails/AdvancedTxDetails';
 import { ChangeFee } from './ChangeFee/ChangeFee';
 import { ReplaceTxButton } from './ChangeFee/ReplaceTxButton';
 import { TxDetailModalBase } from './TxDetailModalBase';
-import { RbfContext, useRbf } from '../../../../../../hooks/wallet/useRbfForm';
+
+const hasRbfParams = (
+    tx: WalletAccountTransaction,
+): tx is RequiredKey<WalletAccountTransaction, 'rbfParams'> => tx.rbfParams !== undefined;
```

Comment:
just realised tiny nit (on my own suggestion, oh well :see_no_evil:  )
This type could be deduplicated. It's two times explicitly here, and also the same thing is declared in `__fixtures__/useRbfForm.ts`  as `type TxWithRequiredRbf`.
That could be moved next to `WalletAccountTransaction` declaration and import it together.

Oh and a fourth occurence in the fixture in `ComposeAndSignFixture` type

Reply:
But abstraction not really needed, explicitly writing the generic makes it very well readable, so I'll leave that up to you :shrug: